setup.py

```diff
@@ -383,6 +383,7 @@ def get_exts_for(name):
         "ipaddress; python_version<'3.7'",
         "envier",
         "pep562; python_version<'3.7'",
+        "opentelemetry-api>=1; python_version>='3.7'",
```

Comment:
We should not assume that the API for setting a context remains the same with the next major version. What happens if the context is not set as expected?

```suggestion
        "opentelemetry-api~=1.0; python_version>='3.7'",
```

Reply:
We’d like to keep the package version as permissive as possible. If we pin this version then our library will fail to install if a customer attempts to use v2.0. The current approach is consistent with opentracing. If v2.0 introduces changes that are incompatible with our library we’ll ask customers to pin their version and we’ll fix the incompatibilities in a patch release.

Reply:
Also backwards compatibility is a strict requirement of the otel api. So v2.0 should not introduce breaking changes: https://opentelemetry.io/docs/reference/specification/versioning-and-stability/#design-goals. Hopefully…..